tests/test_api_v1.py

```diff
@@ -429,7 +431,7 @@ def test_create_request_connection_error(mock_chain, app, auth_env, client, db):
 @mock.patch('flask.send_file')
 @mock.patch('cachito.web.api_v1.Request')
 def test_download_archive(mock_request, mock_send_file, mock_exists, client, app):
-    bundle_archive_path = '/tmp/cachito-archives/bundles/1.tar.gz'
+    bundle_archive_path = os.path.join(tempfile.gettempdir(), 'cachito-archives/bundles/1.tar.gz')
```

Comment:
Might wanna consider using [tmpdir](https://docs.pytest.org/en/latest/tmpdir.html#the-tmpdir-fixture) fixture.